.licenserc.yaml

```
@@ -90,6 +90,7 @@ header:
     - 'thirdparty/fix_libevent_for_macos.patch'
     - 'thirdparty/fix_s2_for_aarch64.patch'
     - 'thirdparty/fix_thrift_for_cpp11.patch'
+    - 'thirdparty/fix_jemalloc_for_m1_on_macos.patch'
```

```suggestion
    - 'thirdparty/fix_jemalloc_for_m1_on_macos.patch'
    - 'thirdparty/fix_libevent_for_macos.patch'
    - 'thirdparty/fix_s2_for_aarch64.patch'
    - 'thirdparty/fix_thrift_for_cpp11.patch'
```
keep in lexicographic order